### PR TITLE
Fix order of DataSetPublishing interval

### PR DESCRIPTION
**Pull request by cristipogacean**
* the Node Publishing Interval configuration prevails now the one at Data Set level
* unit-tests adapted

### CODE REVIEW

**Comment by dacolgit** [comment]
> //  Copyright (c) Microsoft Corporation.  All rights reserved.

no changes in the file

---
Refers to: e2e-tests/IIoTPlatform-E2E-Tests/TestModels/PublishedNodesEntryModel.cs:2 in a769a6b. [](commit_id = a769a6bb9cb153f45348f83a484f9842c4df189b, deletion_comment = False)

**Review by dacolgit — DISMISSED** [review]
:shipit:

File: components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/StandaloneJobOrchestrator.cs [hunk 1]
```diff
@@ -1087,7 +1087,10 @@ IJsonSerializer jsonSerializer
                     OpcAuthenticationUsername = model.OpcAuthenticationUsername,
                     DataSetWriterGroup = model.DataSetWriterGroup,
                     DataSetWriterId = model.DataSetWriterId,
-                    DataSetPublishingInterval = model.DataSetPublishingInterval,
+                    DataSetPublishingIntervalTimespan = model.DataSetPublishingIntervalTimespan,
+                    DataSetPublishingInterval = model.DataSetPublishingIntervalTimespan.HasValue
```
Comment (karok2m) [inline components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/StandaloneJobOrchestrator.cs]: Shouldn't this if/else be the other way around ? Now if `DataSetPublishingIntervalTimespan` has a value, then `DataSetPublishingInterval` would also be assigned a value. And if the first one does not have a value, then value of `model.DataSetPublishingInterval` would be ignored.

Comment (cristipogacean) [reply components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/StandaloneJobOrchestrator.cs]: right ... thanks!

**Review by karok2m — APPROVED** [review]

**Pull request merged**